RF_Receiver/RF_Receiver.ino

```diff
@@ -171,6 +171,8 @@ void setup() {
 	cc1101::setup();
 	#endif
   	initEEPROM();
+	musterDec.MsMoveCount = 0;
```

Comment:
Das Initialisieren sollte besser in der Klasse erfolgen oder was ist der Grund es in setup zu machen?